src/napari_imagej/_function.py

```diff
@@ -424,51 +556,70 @@ def _generate_searchbar(self) -> QLineEdit:
         searchbar.returnPressed.connect(lambda: self._highlight_module(0, 0))
         return searchbar
 
-    def _generate_searcher(self):
+    def _generate_searchers(self) -> List[Any]:
+        searcherClasses = [ModuleSearcher, OpSearcher]
+        resultToModuleInfoConverters = [
+            lambda result: result.info(),
+            lambda result: result.info().cInfo(),
+        ]
         pluginService = ij.get("org.scijava.plugin.PluginService")
-        info = pluginService.getPlugin(ModuleSearcher, Searcher)
-        searcher = info.createInstance()
-        ij.context().inject(searcher)
-        return searcher
+        infos = [pluginService.getPlugin(cls, Searcher) for cls in searcherClasses]
```

Comment:
We should call this `searcherInfo`s, to distinguish from `ModuleInfo`s or `OpInfo`s.

Reply:
I got rid of `info`s entirely, reducing the two list comprehensions to one, in 7b21ee7